### PR TITLE
Add speaker to the speakers list

### PR DESCRIPTION
**Pull request by marianzburlea**

### CODE REVIEW

File: videos/DdYC36N9z0E [hunk 1]
```diff
@@ -0,0 +1,11 @@
+tags:
```
Comment (gandrade) [inline videos/DdYC36N9z0E]: Shouldn't video files have `.yml` extension?

Comment (marianzburlea) [reply videos/DdYC36N9z0E]: Yes, I should

**Comment by marianzburlea** [comment]
All files updated and conflicts resolved. Thanks for your quick reaction!

**Comment by gandrade** [comment]
It looks good now.

**Pull request merged**